hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/common/ManagementEntityState.java

```diff
@@ -13,7 +13,7 @@
 /**
  * Interface for all entity states UI to show the details to a entity.
  */
-public interface ManagmentEntityState<T> {
+public interface ManagementEntityState<Long> {
```

Comment:
You can remove the generic, it's useless:

`public interface ManagementEntityState`